extensions/amp-image-lightbox/amp-image-lightbox.md

```diff
@@ -72,7 +72,7 @@ The `amp-image-lightbox-caption` class is also available to style the caption
 section.
 
 ## Actions
-The `amp-image-lightbox` exposes the following actions you can use [AMP on-syntax to trigger](../../spec/amp-actions-and-events.md):
+The `amp-image-lightbox` exposes the following actions you can use [AMP on-syntax to trigger](https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-events.md):
```

Comment:
SGTM. I assumed the relative links were to make it easy to move entire directories of md files.